query/logicalplan/expr.go

```diff
@@ -243,6 +246,39 @@ func and(exprs []Expr) Expr {
 	}
 }
 
+func Or(exprs ...Expr) Expr {
+	return or(exprs)
+}
+
+func or(exprs []Expr) Expr {
```

Comment:
We could probably also reuse most of this code from the `and` recursive function